fly/commands/ordering_instanced_pipeline.go

```diff
@@ -0,0 +1,58 @@
+package commands
+
+import (
+	"fmt"
+
+	"github.com/concourse/concourse/atc"
+	"github.com/concourse/concourse/go-concourse/concourse"
+
+	"github.com/concourse/concourse/fly/commands/internal/displayhelpers"
+	"github.com/concourse/concourse/fly/commands/internal/flaghelpers"
+	"github.com/concourse/concourse/fly/rc"
+)
+
+type OrderInstancedPipelinesCommand struct {
+	Pipeline     string                         `short:"g" long:"group" required:"true" description:"Name of the instance group"`
+	InstanceVars []flaghelpers.InstanceVarsFlag `short:"p" long:"pipeline" required:"true" description:"Instance vars identifying pipeline to order"`
```

Comment:
This can also be applied to `OrderPipelinesCommand`

```suggestion
	InstanceVars []flaghelpers.InstanceVarsFlag `short:"p" long:"pipeline" required:"true" description:"Instance vars identifying pipeline (can be specified multiple times to provide relative ordering)"`
```